apps/scratch/src/containers/customized-scratch-containers/Blocks.tsx

```diff
@@ -529,9 +533,11 @@ class Blocks extends React.Component<Props, State> {
   }
 
   attachVM() {
-    this.getWorkspace().addChangeListener(
-      suppressStackClicks(this.props.vm.blockListener),
+    // store the wrapped listener so we can remove it later
+    this.wrappedBlockListener = suppressStackClicks(
+      this.props.vm.blockListener,
     );
```

Comment:
Oh my, thanks for catching this and sorry for introducing it in the first place!! 😵

Small suggestion: Instead of defining a class variable, maybe use a class function (analogous to `onWorkspaceChange`)? Then the two use the same pattern and what is similar looks similar? :)